tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -2103,23 +2103,50 @@ def _forward_step(self,
+            else:
+                print(f"[DEBUG] _forward_step - {key}: {type(value)}")
+        
+        if gather_ids is not None:
+            print(f"[DEBUG] _forward_step - gather_ids shape: {gather_ids.shape}, dtype: {gather_ids.dtype}")
+        
         inputs = self._preprocess_inputs(inputs)
+        print(f"[DEBUG] _forward_step - After _preprocess_inputs, keys: {list(inputs.keys())}")
+        
         if inputs.get('spec_metadata', None):
             gather_ids = inputs['spec_metadata'].gather_ids
+            print(f"[DEBUG] _forward_step - Updated gather_ids from spec_metadata: {gather_ids.shape if gather_ids is not None else None}")
```

Ruff (0.12.2)

Line too long (139 > 120)

(E501)